server/index.js

```diff
@@ -95,6 +95,48 @@ app.get('/qa/questions/:question_id/answers', (req, res) => {
     });
 });
 
+// post an answer
+app.post('/qa/questions/:question_id/answers', (req, res) => {
+  axios.post(req.url, req.body)
+    .then(() => {
+      console.log(req.body);
+      res.status(201).send('CREATED');
+    })
+    .catch((err) => {
+      res.status(504).send('error in posting new answer', err);
+    });
+});
+
+// put helpfulness for a question
+app.put('/qa/questions/:question_id/helpful', (req, res) => {
+  axios.put(req.url)
+    .then(() => {
+      res.status(204).send('CREATED');
+    })
+    .catch((err) => {
+      res.status(504).send('error in putting question as helpful', err);
+    });
+});
+
+// put helpfulness for an answer
+app.put('/qa/answers/:answer_id/helpful', (req, res) => {
+  axios.put(req.url)
+    .then(() => {
+      res.status(204).send('NO CONTENT');
+    })
+    .catch((err) => {
+      res.status(504).send('error in putting answer as helpful', err);
+    });
+});
+
+// put reported for an answer
+app.put('/qa/answers/:answer_id/report', (req, res) => {
+  axios.put(req.url)
+    .then(() => {
+      res.status(204).send('NO CONTENT');
+    })
+    .catch((err) => {
+      res.status(504).send('error in putting answer as reported', err);
```

Comment:
@afong888 I think this put request got mixed in with the following get request and my server is not working. Can you take a look?